src/modules/IscsiClientLib.rb

```diff
@@ -751,7 +751,15 @@ def getCurrentNodeValues
     # @return  [Bool]    nodes are equal?
     #
     def equalNodes?(n1, n2)
-      return false if n1.empty?
+      return false if n1.empty? || n2.empty?
```

Comment:
Can be possibly *both* empty?

Reply:
Well, at least **either** might. I don't think it's fair to assume it's the 1st.